spec/unit/typesafe/config/conf_parser_spec.rb

```diff
@@ -781,20 +781,26 @@ def assert_comments_at_path_index(comments, conf, path, index)
   # Skipping 'includeURLHeuristically' because we don't support URLs
   # Skipping 'includeURLBasenameHeuristically' because we don't support URLs
 
-  it "acceptBOMStartingFile" do
-    skip("BOM not parsing properly yet; not fixing this now because it most likely only affects windows") do
-      # BOM at start of file should be ignored
-      conf = Hocon::ConfigFactory.parse_file(TestUtils.resource_file("bom.conf"))
-      expect(conf.get_string("foo")).to eq("bar")
-    end
+  it "doesnotacceptUTF8FileNames" do
+    expect { Hocon::ConfigFactory.parse_file(TestUtils.resource_file("ᚠᛇᚻ.conf")) }.to raise_error
   end
```

Comment:
I don't have a strong opinion; your logic makes sense.  I think I might be inclined to at least throw a comment in here, though, explaining that this is not the desired behavior.  In the absence of that, in my mind, this test reads like an assertion that we don't *want* to support UTF8 file names.  It's not a huge deal, though.